code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1728,3 +1728,11 @@
 /datum/reagent/tranquility/reaction_mob(mob/living/L, method=TOUCH, reac_volume, show_message = 1, touch_protection = 0)
 	if(method==PATCH || method==INGEST || method==INJECT || (method == VAPOR && prob(min(reac_volume,100)*(1 - touch_protection))))
 		L.ForceContractDisease(new /datum/disease/transformation/gondola(), FALSE, TRUE)
+
+
+/datum/reagent/spider_extract
+	name = "Spider Extract"
+	description = "A highly specialized extract coming from the Australicus sector, used to create broodmother spiders."
+	color = "#ED2939"
+	taste_description = "upside down"
+	can_synth = FALSE
```

Comment:
awwwwww

C'mon, let a joint operation between botany, xenobiology, and Lady Luck let people make some non-antag broodmother spiders (without the involvement of a traitor) once in a blue moon.